RRS/accounts/views.py

```diff
@@ -1,3 +1,28 @@
+from django.http import HttpResponseRedirect, HttpResponse
 from django.shortcuts import render
+from accounts import forms
+from django.contrib.auth import authenticate
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
 
-# Create your views here.
+from staff import utils
+
+   
+def signup(request):
+    context = {}
+    if request.method == "GET":
+        form = forms.SignUpForm()
+        print form
+    if request.method == "POST":
+        form = forms.SignUpForm(request.POST)
+        if form.is_valid():
+            user = User.objects.create_user(form.cleaned_data["username"], form.cleaned_data["email"], 
+                                            form.cleaned_data["password1"])
+            user.first_name = form.cleaned_data["first_name"]
+            user.last_name = form.cleaned_data["last_name"]
+            user.save()
+            return HttpResponseRedirect(reverse('index'))
+        else:
```

Comment:
You can remove the else statement by using [guard clauses] (http://sourcemaking.com/refactoring/replace-nested-conditional-with-guard-clauses)